onionshare_gui/onionshare_gui.py

```diff
@@ -132,13 +132,27 @@ def __init__(self, onion, qtapp, app, filenames, config=False):
         # Status bar
         self.status_bar = QtWidgets.QStatusBar()
         self.status_bar.setSizeGripEnabled(False)
-        self.status_bar.setStyleSheet("QStatusBar::item { border: 0px; }")
+        statusBar_cssStyleData ="""
+        QStatusBar {
+            font-style: italic;
+            color: #666666;
+        }
+
+        QStatusBar::item {
+            border: 0px;
+        }"""
+
+        self.status_bar.setStyleSheet(statusBar_cssStyleData)
         self.status_bar.addPermanentWidget(self.server_status_indicator)
         self.status_bar.addPermanentWidget(self.settings_button)
         self.setStatusBar(self.status_bar)
 
         # Status bar, zip progress bar
         self._zip_progress_bar = None
+        # Status bar, other larger messages
+        self._close_on_timeout_label = None
+        self._closing_automatically_label = None
+        self._timeout_download_still_running_label = None
```

Comment:
Is it necessary for this to be three separate labels? Could all three of these things just update the same label? This way, if we have more messages like this in the future, we don't need to continue to add more labels.

Reply:
Good call - I don’t know why I didn’t do it that way :)